### DISCUSSION TITLE
does canary version crash?

### DISCUSSION THREAD

**Discussion by Leo-camp** [Gniang/RoundedTB#17] [Q&A]
i have been using roundtb for 2 years , but after windows latest upadte it stop working . 

 so i have been found this canary version , is it buggy or crash in between .
i have been using laptop for work purposes so good to know , 
if anyone has tried it for a week or two after latest upadte,
please review about it

**Comment by forditohu** [comment]
I'm also very much awaiting a working version after Windows update 23H2, but the Canary version (231016_2) only starts and crashes for me within a couple of seconds. It doesn't get to the point of making any visible change on the taskbar.
